### PR TITLE
Adds min/max sale price querying/filtering to the bike page

### PR DESCRIPTION
**Pull request by nfourtythree**

### CODE REVIEW

File: modules/Module.php [hunk 1]
```diff
@@ -103,8 +104,8 @@ function($event) {
         Event::on(
             Entry::class,
             Entry::EVENT_AFTER_DELETE,
-            function($event) {
-                /** @var Entry $entry */
+            function(Event $event) {
```
Comment (timkelty) [inline modules/Module.php:107]: I fixed this. Only some kind of _sane_ person would assume `EVENT_AFTER_SAVE` and `EVENT_AFTER_DELETE` receive the same kind of event.

**Review by timkelty — CHANGES_REQUESTED** [review]
CleanShot 2024-09-16 at 13 17 08@2x

"clear filters" doesn't appear to work with the new price field. I tried to add it quickly, but failed. Up to you if you think it matters or not, but I noticed it so passing it on.

File: modules/Module.php (hunk 1 above)
Comment (nfourtythree) [reply modules/Module.php:107]: 😂

**Comment by nfourtythree** [comment]
> CleanShot 2024-09-16 at 13 17 08@2x
> 
> "clear filters" doesn't appear to work with the new price field. I tried to add it quickly, but failed. Up to you if you think it matters or not, but I noticed it so passing it on.

Have pushed a fix for this, everything should be ready to go if you are happy to merge it

**Pull request merged**